backend/api/src/main/kotlin/io/tolgee/hateoas/contentDelivery/ContentDeliveryConfigModel.kt

```diff
@@ -20,6 +20,7 @@ class ContentDeliveryConfigModel(
   val publicUrl: String?,
   val autoPublish: Boolean,
   val lastPublished: Long?,
+  val files: Collection<String>,
```

Comment:
This is confusing name. Would future reader understand that these are the files which being published in past and not files to publish? 

```suggestion
  val lastPublishedFiles: Collection<String>,
```